packages/lodestar/src/chain/blocks/importBlock.ts

```diff
@@ -86,9 +86,15 @@ export async function importBlock(chain: ImportBlockModules, fullyVerifiedBlock:
   // current justified checkpoint should be prev epoch or current epoch if it's just updated
   // it should always have epochBalances there bc it's a checkpoint state, ie got through processEpoch
   const justifiedCheckpoint = postState.currentJustifiedCheckpoint;
-  const onBlockPrecachedData: OnBlockPrecachedData = {executionStatus};
+  const state = getStateForJustifiedBalances(chain, postState, block);
+
+  const onBlockPrecachedData: OnBlockPrecachedData = {
+    executionStatus,
+    justifiedActiveValidators: state.totalActiveValidators,
+    justifiedTotalActiveBalanceByIncrement: state.totalActiveBalanceByIncrement,
```

Comment:
This is an optimization point, could move justifiedActiveValidators, justifiedTotalActiveBalanceByIncrement as private members of forkchoice and update same like justifiedBalances.

Reply:
What is this optimization for specifically? To speed up which function and by how much? Please document and justify as much as possible any changes that add complexity

Reply:
This is done for computing the score where total active balance, and total total active validators, however could just iterate and count the `justifiedBalances` like how lighthouse does in their compute scope fn

Reply:
reverting this optimization and directly calculating these values inside computeProposerBoostScoreFromBalances as the cost is insignificant:
```
✓ computeProposerBoostScoreFromBalances                               6907.508 ops/s    144.7700 us/op        -     206061 runs   30.0 s
```